pegasus/sites.v3/code.org/public/js/csf-congrats.js

```diff
@@ -30,7 +27,6 @@ $(document).ready(function()
 });
 
 var petition;
-$(document).ready(function()
-{
+$(document).ready(function() {
```

Comment:
Did this file change at all, or just get Prettier run on it?

Reply:
just prettier. somewhat on accident, but I decided to leave it since this now seems to match our preferred style.